### PR TITLE
fix(amplication-server): validate numbers min/max to ensure min is not greater than max, fixes #515

### LINKED ISSUE #515: Client: numaric max/min values are not coordinated
the entitys numaric max/min values are not coordinated
therfore the user can set a min val greater then the max

### PR DESCRIPTION
**Pull request by MichaelSolati**
Issue Number: #515

## PR Details

Validate that `minimumValue` is not greater than, or equal to, `maximumValue`.

## PR Checklist
- [ x] Tests for the changes have been added
- [ x] `npm test` doesn't throw any error

### CODE REVIEW

**Comment by yuval-hazaz** [comment]
well done @MichaelSolati 👏

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
looks good to me
@MichaelSolati I think you should add a test to make it perfect

File: package-lock.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "amplication",
```
Comment (tupe12334) [inline package-lock.json:7]: Wich node and npm versions are you working with?

**Comment by tupe12334** [comment]
@MichaelSolati Pay attention to the branch name we usually use the issues number as this format 
<type>/<issus number>-<title>
fix/515-schema-validation

**Comment by MichaelSolati** [comment]
> @MichaelSolati Pay attention to the branch name we usually use the issues number as this format /-<title> fix/515-schema-validation

Noted for next time

File: package-lock.json (hunk 1 above)
Comment (tupe12334) [reply package-lock.json:7]: @MichaelSolati ? I don't see why you added this property.

**Comment by yuval-hazaz** [comment]
@MichaelSolati @tupe12334 what is the status of this PR?

File: package-lock.json (hunk 1 above)
Comment (tupe12334) [reply package-lock.json:7]: @MichaelSolati Can you please remove this change? or let me know if I can

**Comment by arielweinberger** [comment]
Since there hasn't been a response in over a month, I am going to close this.

**Pull request closed without merging**